frontend/src/views/RequestGame.vue

```diff
@@ -89,16 +89,58 @@ export default defineComponent({
 		let newMessage: NewMessage = reactive({
 			sender_id: Number(0),
 			chat: Number(props.chat_id),
+			is_game_request: true,
 			content: 'wanna play PONG?',
 		});
 		return {
 			gameStore,
-			errors,
-			createGame,
 			allUsers,
 			createGameForm,
 			newMessage,
 		};
 	},
+	data() {
+		return {
+			errors: [] as String[],
```

Comment:
just a question, why did you moved the errors? *if this takes to much time to reply on e can discuss this later!